spinta/components.py

```diff
@@ -751,11 +751,12 @@ def get_from_flatprops(self, prop: str) -> Property:
 
 
 class PropertyGiven:
-    access: str = None
-    enum: str = None
-    unit: str = None
-    name: str = None
+    access: Optional[str] = None
```

Comment:
siūlyčiau rašyti ne `Optional[str]` bet `str | None`, t.y. naudoti python in-built tipų aprašymo priemones, nes visos `spinta` palaikomos versijos tai palaiko.

Reply:
Spinta palaiko Python 3.9 ir aukštesnes versijas:
https://github.com/atviriduomenys/spinta/blob/3d1bfee2d00bb7d0fa1d02769bde75e87f7e137f/pyproject.toml#L36-L37

Built-in python tipai pristatyti tik Python 3.10 versijoje:
> Changed in version 3.10: Optional can now be written as X | None.

* https://docs.python.org/3/whatsnew/3.10.html#pep-604-new-type-union-operator.
* https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
tada gal geriau naudoti `from __future__ import annotations`. Bet aš jau pradedu maišytis `spinta` palaikomose versijose - man atrodo, kad nors čia ir parašyta `3.9`, bet ji jau nebepalaikoma paties `Python` (tiksliau, deprecated ir greitai bus visai nebvepalaikoma), tai nuspręsta, kad ir `spinta` jos turėtų nebepalaikyti. 

Dabar nerandu, kur visa tai nurodyta, bet bent jau čia rašoma, kad reikalinga Python 3.10 ar aukštesnė versija: https://spinta.readthedocs.io/en/latest/contrib/setup.html

Reply:
aha, matau ir github actions nurodoma, kad žemiausia versija 3.9 - tai gal vis dėlto ji dar palaikoma. 

Bet vis tiek, man atrodo, kad pridėjus `from__future__ import annotations` turėtų ir 3.9 versijoje leisti `str | None`, tai mano nuomone, čia būtų patogiausias variantas.

Reply:
[3a7015d](https://github.com/atviriduomenys/spinta/pull/1118/commits/3a7015df51c4edd21ccffb5c237ffbbbc352107b)